packages/astro/src/core/build/index.ts

```diff
@@ -53,6 +55,42 @@ export interface BuildOptions {
 	force?: boolean;
 }
 
+function resolveInjectedAssets({
+	settings,
+	cwd,
+}: {
+	/** Astro Settings object */
+	settings: AstroSettings;
+	/** Current working directory */
+	cwd?: string;
+}) {
+	const { config } = settings;
+	for (const injectedAsset of settings.injectedAssets) {
+		const { entrypoint } = injectedAsset;
+		let resolved: string;
+		try {
+			resolved = require.resolve(entrypoint, { paths: [cwd || fileURLToPath(config.root)] });
```

Comment:
Hmm… I think we want to resolve these relative to the integration rather than the cwd or project root? That’s my understanding at least.